src/apolloQueryObservable.ts

```diff
@@ -23,26 +21,19 @@ export class ApolloQueryObservable<T> extends Observable<T> {
 
   // apollo-specific methods
 
-  public refetch(variables?: any): Promise<any> {
+  public refetch(variables) {
     return this.apollo.refetch(variables);
   }
 
-  public stopPolling(): void {
+  public stopPolling() {
     return this.apollo.stopPolling();
   }
 
-  public startPolling(p: number): void {
+  public startPolling(p) {
```

Comment:
I thought that `IObservableQuery` will take care of types but it seems I was wrong